server/models/Order.ts

```diff
@@ -647,6 +650,23 @@ Order.prototype.unPause = async function (user: User, { UserTokenId = undefined
   return this;
 };
 
+/**
+ * Marks all other paused orders from the same fromCollective/collective as cancelled.
+ */
+Order.prototype.markSimilarPausedOrdersAsCancelled = async function (): Promise<void> {
+  await models.Order.update(
+    { status: OrderStatus.CANCELLED },
+    {
+      where: {
+        id: { [Op.not]: this.id },
+        FromCollectiveId: this.FromCollectiveId,
+        CollectiveId: this.CollectiveId,
+        status: OrderStatus.PAUSED,
+      },
+    },
+  );
```

Comment:
Looks good. So, no need to do anything else than updating the model?

Reply:
Nope, the subscription itself has already been deactivated.